svn/common.py

```diff
@@ -73,7 +54,7 @@ def run_command(self, subcommand, args, success_code=0,
 
         environment_variables = os.environ.copy()
         environment_variables['LANG'] = 'en_US.UTF-8'
-        environment_variables.update(self._env)
+        environment_variables.update(self.__env)
```

Comment:
Do this before the "LANG" assignment and you have a deal. Since that's an explicit parameter and the environment collection just provides support for something that is general in nature, the latter can't be allowed to potentially overwrite the former. Thanks.